core-bundle/src/Resources/contao/library/Contao/Template.php

```diff
@@ -399,6 +399,46 @@ public function trans($strId, array $arrParams=array(), $strDomain='contao_defau
 		return System::getContainer()->get('translator')->trans($strId, $arrParams, $strDomain);
 	}
 
+	/**
+	 * Helper method to allow quick access in the Contao templates for raw output.
+	 * Be careful when using this. It must NOT be used within regular HTML when $value
+	 * is uncontrolled user input. It's useful to ensure raw values within e.g. <code> examples
+	 * or JSON-LD arrays.
+	 */
+	public function raw(string $value): string
+	{
+		return StringUtil::revertInputEncoding($value);
+	}
+
+	/**
+	 * Helper method to allow quick access in the Contao templates for raw output.
+	 *
+	 * Compared to $this->raw() it does remove HTML.
+	 *
+	 * Be careful when using this. It must NOT be used within regular HTML when $value
+	 * is uncontrolled user input. It's useful to ensure raw values within e.g. <code> examples
+	 * or JSON-LD arrays.
+	 */
+	public function rawPlainText(string $value, bool $removeInsertTags = false): string
```

Comment:
I'd prefer `rawText` and `rawHtml` method names

Reply:
`rawHtml` would be very confusing IMO as it does not output HTML.